biosdk-services/src/main/java/io/mosip/biosdk/services/config/LoggerConfig.java

```diff
@@ -13,24 +12,8 @@ private LoggerConfig() {
 
 	}
 
-	
-	/** The mosip rolling file appender. */
-	private final static RollingFileAppender MOSIP_ROLLING_APPENDER = new RollingFileAppender();
-	
-	static {
-        MOSIP_ROLLING_APPENDER.setAppend(true);
-        MOSIP_ROLLING_APPENDER.setAppenderName("org.apache.log4j.RollingFileAppender");
-        MOSIP_ROLLING_APPENDER.setFileName("./logs/biosdk-service.log");
-        MOSIP_ROLLING_APPENDER.setFileNamePattern("./logs/biosdk-service-%d{yyyy-MM-dd}-%i.log");
-        MOSIP_ROLLING_APPENDER.setImmediateFlush(true);
-        MOSIP_ROLLING_APPENDER.setMaxFileSize("50mb");
-//		mosipRollingFileAppender.setMaxHistory(3);
-        MOSIP_ROLLING_APPENDER.setPrudent(false);
-//		mosipRollingFileAppender.setTotalCap("50mb");
-	}
-	
 	public static Logger logConfig(Class<?> clazz) {
-		return Logfactory.getDefaultRollingFileLogger(MOSIP_ROLLING_APPENDER, clazz);
+		return Logfactory.getSlf4jLogger(clazz);
 	}
 
```

Comment:
As per https://mosip.atlassian.net/browse/MOSIP-29287, I don't see any new logs added for debugging.

Reply:
```suggestion
It is already added in Previous PRs. But it was not showing due the issue in the Logger Factory. This PR contains that fix only.
```